src/main/java/use_cases/display_availability_timeblocks/DisplayAvailabilityTimeBlockModel.java

```diff
@@ -0,0 +1,11 @@
+package use_cases.display_availability_timeblocks;
+import java.time.LocalDateTime;
+
+public class DisplayAvailabilityTimeBlockModel {
+
+    public DisplayAvailabilityTimeBlockModel() {
+
```

Comment:
Make sure you remember to implement this! As it stands you won't get errors, but you wanna make sure you're writing clean and applicable code at all steps.